media-proxy/src/proxy_context.cc

```diff
@@ -570,7 +570,7 @@ int ProxyContext::RxStart_mtl(const mcm_conn_param *request)
     int ret;
 
     /*add lock to protect MTL library initialization to aviod being called by multi-session simultaneously*/
-    if (!mDevHandle && imtl_init_preparing == false) {
+    if (mDevHandle == NULL && imtl_init_preparing == false) {
```

Comment:
Why this change? We usually use !val to check if value is NULL.

Reply:
To align with code in TxStart_mtl()

Reply:
> To align with code in TxStart_mtl()

Could you elaborate on that?

Reply:
I guess we just have to define our approach. In a new code we tend to use `!val`, in old code there is a mix of everything.
Let's agree on something. Me personally I'm okay with both approaches. `val == NULL` is slightly more readable, `!val` is smaller and more elegant.

Reply:
I vote for `if (!something) {}`.

Reply:
I also vote for ```!val```

Reply:
I declare to all and sundry that the winner is `!val` :tada:

Reply:
Removed this change